bazel/BUILD

```diff
@@ -111,11 +111,19 @@ cc_library(
         "-Wformat",
         "-Wformat-security",
         "-fstack-protector-all",
+        "-march=native",
```

Comment:
Since we are copying the same flags everywhere, please use a Bazel function to define these and then call them wherever they are needed.